src/Symfony/Bundle/WebProfilerBundle/Controller/ExceptionController.php

```diff
@@ -105,7 +105,7 @@ public function cssAction($token)
 
     protected function getTemplate()
     {
-        return '@Twig/Exception/'.($this->debug ? 'exception' : 'error').'.html.twig';
+        return '@Twig/Exception/'.($this->debug ? 'exception_full' : 'error').'.html.twig';
```

Comment:
Note that `error.html.twig` (that is used when `debug` is `false`) contains HTML page too.